guide/designing-products/user-research-resources.md

```diff
@@ -37,29 +37,53 @@ Below is a collection of published user research findings around Bitcoin, from a
 ## Research papers
 
 ###### [Non-adoption of crypto-assets: Exploring the role of trust, self-efficacy, and risk](https://voskart.de/pdf/crypto_adoption.pdf)
+
+*A survey of 404 users and non-users of Bitcoin about the roles of trust, self-efficacy, and risk on adoption intention.*
+
 Published in May 2021 by Artemij Voskobojnikov, Svetlana Abramova, Konstantin Beznosov, and Rainer Böhme.
```

Comment:
Could minimize the amount of text here by formatting these details like below. Would also recommend italicizing this content as it is less emphatic and gives more emphasis to the summary (the maint content people will be focused on). See my comment above about removing italics from the summary also. 

```suggestion
_Published May 2021 by Voskobojnikov, Abramova, Beznosov, and Böhme._
```